distribution/docs/src/main/resources/content/_architectures/url-resource-reader.adoc

```diff
@@ -31,6 +31,25 @@ Configure the URL Resource Reader from the ${admin-console}.
 
 See <<ddf.catalog.resource.impl.URLResourceReader,URL Resource Reader configurations>> for all possible configurations.
 
+=== Configuring Permissions for the URL Resource Reader
+
+Configuring the URL Resource Reader to retrieve files requires adding permissions to the Security Manager.
+
+Configuring the URL Resource Reader requires adding read permissions to the directory containing the resources. The following permissions, replacing <DIRECTORY_PATH> with the path of the directory containing resources should be placed in the URL Resource Reader section inside ${home_directory}/security/configurations.policy.
+
+.Adding New Permissions
+[WARNING]
+====
+After adding permissions, a system restart is required for them to take effect.
+====
+
+. permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
+. permission java.io.FilePermission "<DIRECTORY_PATH>${/}-", "read";
+
+Trailing slashes after <DIRECTORY_PATH> have no effect on the permissions granted. For example, adding a permission for "/test/path" and "/test/path/" are equivalent. The recursive forms "/test/path${/}-", and "/test/path/${/}-" are also equivalent.
```

Comment:
❓ Should we consider having a section that describes the rules related to adding permissions instead of repeating them? We have the exact same sentence for CDM and will probably have more to add in the future, so having these in a common section and referring to it would ensure instructions don't diverge over time.

Reply:
Yes, @lessarderic I struggled with this from an ease of reading and maintainability perspective. @ricklarsen do you have a thought on this one. I tried to make it clear for the Resource Reader case in this section, so a reader would not have to jump around sections. For example, they only use the Resource Reader but not a CDM. However, it would be better to reference a common section from a maintenance perspective.

Reply:
I think it's probably ok to have a few details repeating (like you say, to keep from bouncing people all over) but if we do start finding large chunks of material need to be repeated it's probably good to have a centralized section. We could still have a sentence or two to cover the majority of uses and say "for more detail, see `<<_details...`

Reply:
All right I'll leave it the same for now. Probably worth reconsidering as we add more permissions docs.